src/components/autocomplete/index.tsx

```diff
@@ -56,6 +56,11 @@ interface AutocompleteProps extends Omit<InputGroupProps, 'onChange'> {
   onChange: (value: string) => void;
   onInputChange?: (e: React.ChangeEvent<HTMLInputElement> | string) => void;
   isFromTransactions?: boolean;
+  actionOnFocus?: (value?: any) => void;
```

Comment:
remove optional of params: actionOnFocus, actionOnSelect, actionOnRemoveInput, actionOnBlur